content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md

```diff
@@ -30,38 +30,38 @@ In the tutorial, you will first make a workflow file that uses the [`peter-evans
 1. Copy the following YAML contents into your workflow file.
 
     ```yaml copy
-    {% data reusables.actions.actions-not-certified-by-github-comment %}
-
-    {% data reusables.actions.actions-use-sha-pinning-comment %}
-
     name: Add comment
     on:
       issues:
         types:
           - labeled
     jobs:
       add-comment:
-        if: github.event.label.name == 'help-wanted'
+        if: github.event.label.name == 'help wanted'
         runs-on: ubuntu-latest
         permissions:
           issues: write
         steps:
           - name: Add comment
-            uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
-            with:
-              issue-number: {% raw %}${{ github.event.issue.number }}{% endraw %}
-              body: |
-                This issue is available for anyone to work on. **Make sure to reference this issue in your pull request.** :sparkles: Thank you for your contribution! :sparkles:
+            # https://cli.github.com/manual/gh_issue_comment
+            run: gh issue comment -R "$REPOSITORY" "$NUMBER" --body "$BODY"
```

Comment:
Seems a bit "excessive" to create an environment variable only to use it here, once, directly. 
What do you think about simplifying to 
```
run: gh issue comment ${{ github.event.issue.number }} --repo ${{ github.repository }} --body "$BODY"
```
Also, note the removal of quotation marks given that an issue number will not have a space in it. Neither will the `github.repository`. 

And the order when you run `gh issue comment --help` is this:
```
USAGE
  gh issue comment {<number> | <url>} [flags]
```

So, we mention the issue number first and then all flags together after.

Reply:
imo with all that one one line its hard to read, especially on a phone screen. so you're left with something like this:

```yml
run: |
  gh issue comment ${{ github.event.issue.number }} \
    --repo ${{ github.repository }} --body "$BODY"
```

idk. i just put it in env vars so the `run: ...` line was one line. it also had the added benefit of feeling more like a `with: body: ...` input on a normal action 🤷‍♂️

Reply:
We typically do try to use variables to avoid injection attacks, even though it is more overhead/more verbose. An injection attack is not really a risk here, but it gets people in the habit of coding defensively.